### ISSUE TITLE
ideas for minor usability improvements

### ISSUE THREAD

**Issue by aslgraefe** [Phoneboard/Phoneboard#111]
In here I'll "dump" all ideas to improve phenoboard's usability. Using version v0.5.117
- in HPO text mining
  - when clicking on a mined term within the text, the window that shows "Label: xy. TermID: xy Original: cy" is quite far away from the actual term. Putting it closer feels more intuitive. Also, for terms on the far right of the text this window is not shown. 
  - when adding new terms into the list, it would be nice to kind of move the new terms further up in the list so that I can organise the terms by onset time. This may be helpful for very long clinical descriptions to sort the case by longitudinal onset times.
  - wenn adding onset for a term, a small explanation window or link to the [onset term ](https://hpo.jax.org/browse/term/HP:0003674) that includes all children and respective age defintions could be helpful
  - in the case I am curating right now, we have the same phenotype twice at age 7 and at age 10. But it is not possible to add two onsets or the same term twice. How is this going to be handled? 
- template editor view:
  - choosing between (1) and (2) for each variant in the allels column is not explained. Also, clicking the "+" does not work (at first it showed two windows, then none at all).
